ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -567,6 +596,89 @@ static enum pkcs11_rc create_data_attributes(struct obj_attrs **out,
 					  ARRAY_SIZE(raw_data_opt_or_null));
 }
 
+static enum pkcs11_rc create_certificate_attributes(struct obj_attrs **out,
+						    struct obj_attrs *temp)
+{
+	uint32_t const *mandated = NULL;
+	uint32_t const *optional = NULL;
+	size_t mandated_count = 0;
+	size_t optional_count = 0;
+	uint32_t attr_size = 0;
+	uint32_t cert_category = PKCS11_CK_CERTIFICATE_CATEGORY_UNSPECIFIED;
```

Comment:
Should maybe be renamed `default_category` to avoid confusion.

Same a next line: `name_hash_alg`  => `default_hash_alg`.

Reply:
OK